incubator/java-spring-boot2/stack.yaml

```diff
@@ -16,7 +16,10 @@ requirements:
    docker-version: ">= 17.09.0"
 templating-data:
   baseimage: maven:3.6-ibmjava-8
+  baseextrapkgs: curl wget xmlstarlet unzip ca-certificates
```

Comment:
I am having second thoughts about how much is in this string. If xmlstartlet is removed, the whole stack will break. We use it everywhere. I'm not sure this should be moved out this far...